components/artwork_filter_2/models/counts.coffee

```diff
@@ -0,0 +1,55 @@
+Backbone = require 'backbone'
+_ = require 'underscore'
+metaphysics = require '../../../lib/metaphysics.coffee'
+aggregationsMap = require '../aggregations_map.coffee'
+query = require '../queries/counts.coffee'
+
+module.exports = class Counts extends Backbone.Model
+  initialize: ({ @params }) ->
+    @listenToOnce @params, 'firstSet', @fetch
+    _.each @params.filterParamKeys, (param) =>
+      @listenTo @params, "change:#{param}", @setCurrentCounts
+
+  aggregations: ['TOTAL', 'GALLERY', 'INSTITUTION', 'PERIOD', 'MEDIUM']
+
+  # Structures the total counts and aggregations from metaphysics,
+  # for both `for_sale` and for all artworks, for easier re-rendering
+  mapData: ({ all, for_sale }) ->
```

Comment:
Is this totally necessary? Even if it is, this seems like a better job for Metaphysics than localizing this logic.